lib/std/ioutils.nim

```diff
@@ -0,0 +1,62 @@
+when defined(windows):
+  proc c_dup(oldfd: FileHandle): FileHandle {.
+    importc:"_dup", header: "<io.h>".}
+  proc c_dup2(oldfd: FileHandle, newfd: FileHandle): cint {.
+    importc: "_dup2", header: "<io.h>".}
+else:
+  proc c_dup(oldfd: FileHandle): FileHandle{.
+    importc: "dup", header: "<unistd.h>".}
+  proc c_dup2(oldfd: FileHandle, newfd: FileHandle): cint {.
+    importc: "dup2", header: "<unistd.h>".}
+
+const SupportIoctlInheritCtl = (defined(linux) or defined(bsd)) and
+                              not defined(nimscript)
+when SupportIoctlInheritCtl:
+  var
```

Comment:
* s/var/let/ since it's a C constant (it's a new-ish feature that allows importc let without a value since it's defined in C)
* it's neither used not exported, either use it or export it disable with `when false` for future un-disabling